### DISCUSSION TITLE
validation set accuracy

### DISCUSSION THREAD

**Discussion by saar-eliad** [google-research/text-to-text-transfer-transformer#824] [Q&A]
Hi, 
after switching PyTorch version + using an updated model from huggingFace after they fixed a small bug,
I noticed a small 0.5% drop in our validation set accuracy.
I want to compare it to your numbers - however, I did not find your validation set results.
Are they available somewhere?
 
Can you share them for T5-3B super-glue: WIC, BoolQ, MultiRC, RTE?
(the results I'm getting are very close to what you published on the test set).
